network-api/networkapi/wagtailpages/tests/research_hub/test_author_index.py

```diff
@@ -151,17 +151,15 @@ def test_get_author_detail_context_multiple_detail_pages(self):
         )
 
         # author, locale, detail pages.
-        with self.assertNumQueries(3):
-            context = self.author_index.get_author_detail_context(
-                profile_id=self.research_profile.id,
+        with self.assertNumQueries(2):
```

Comment:
Hi @tbrlpld! 

I updated this because once we updated the method to use "get_latest_research" instead of the get_author_context, the number of queries seem to have dropped from 3 to 2? 

I was wondering if you could double check if this is correct, other than that, no problems adding tests for the breadcrumbs!